_docs/resources/index.md

```diff
@@ -1,7 +1,111 @@
 ---
 title: Resources
-layout: categories
+layout: family
 category: Resources
 ---
 
-Resources are materials and information used for reference.
+<section class="static-section" markdown="1">
+
+## Sketch Resources
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/sketch-library.svg" alt="designer UX icon"/>
```

Comment:
All of these use the same alt text. I recommend updating them to be more accurate.

Reply:
updated